tests/main/connect-undo/test-connect.v1/bin/consumer

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec nc -h > /dev/null
```

Comment:
It seems that nethack differs across distributions. Probably there are different implementations selected via the alternatives system.

Reply:
Ah, that's annoying, I'll check it out, thanks.

Reply:
Nb, netcat, not nethack 😄

Reply:
I've tried a couple of things including moving away from netcat and use a local service + python script (based on interfaces-network test), but in all cases it would need exceptions (e.g. due to lack of strict confinement). I think it's beyond the point of this test to try to run it everywhere, so restricted it to ubuntu to keep it simple.